### PR TITLE
Integrate gutenberg-mobile release 1.60.1

### PR DESCRIPTION
**Pull request by geriux**
## Description
This PR incorporates the 1.60.1 release of gutenberg-mobile.
For more information about this release and testing instructions, please see the related Gutenberg-Mobile PR: https://github.com/wordpress-mobile/gutenberg-mobile/pull/3872

Release Submission Checklist

- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

File: Podfile [hunk 1]
```diff
@@ -166,7 +166,7 @@ abstract_target 'Apps' do
     ## Gutenberg (React Native)
     ## =====================
     ##
-    gutenberg :tag => 'v1.60.0'
+    gutenberg :commit => 'd59918b9aadcf958ce7b8a98ee636d571e58773f'
```
Comment (dcalhoun) [inline Podfile]: Friendly reminder that we will need to update this to the GB Mobile tag after we create it.

**Review by dcalhoun — APPROVED** [review]
LGTM! Thank you! Provided the CI jobs pass, we should be good to go. 👍🏻

File: Podfile (hunk 1 above)
Comment (geriux) [reply Podfile]: Updated, thanks! 🙌

**Comment by mokagio** [comment]
@geriux  this has been bundled as part of [18.1 beta 3 (18.1.0.3)](https://github.com/wordpress-mobile/WordPress-iOS/pull/17094).

Thanks for your work 🙌

**Pull request merged**